sps1112/test-rpg
# Update Headers and Game States

## **Pull Request Template**

- No linked issue
- Updated headers, game states and battle format

### _Steps to test_

To test PR:-

1. Clone Repo and build using CMake
1. Test using Bash, Powershell or the .exe in build/Debug/
1. The game should play out without any bugs

### _Things to Know_

No such info
